app/controllers/works_controller.rb

```diff
@@ -358,17 +368,17 @@ def update
     @work.set_challenge_claim_info
     set_work_form_fields
 
-    if params[:edit_button] || work_cannot_be_saved?
-      set_work_tag_error_messages
-      render :edit
-    elsif params[:preview_button]
+    if params[:preview_button]
```

Comment:
I don't think you can reorder the conditions here, at least not without more extensive changes. If someone enters invalid inputs and presses preview, they're not supposed to see a preview with those invalid inputs. Instead, they should be bounced back to the edit form with errors showing what they need to fix.

Reply:
Got it, changed conditions order back.